packages/components/src/test/tree/controlled/UsePresentationTreeState.test.ts

```diff
@@ -108,6 +108,15 @@ describe("usePresentationTreeState", () => {
     await waitFor(() => expect(result.current?.nodeLoader).to.not.eq(oldNodeLoader));
   });
 
+  it("creates new reporting nodeLoader when `onNodeLoaded` callback passed", async () => {
```

Comment:
Does not look like this test actually verifies that reporting nodeLoader is created. Looking at previous test it I would  assume that in this test new provider was created because of `pageSize` and `onNodeLoader` probably didn't have any effect.